CHANGELOG.md

```diff
@@ -4,7 +4,28 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project attempts to match the major and minor versions of [stactools](https://github.com/stac-utils/stactools) and increments the patch number as needed.
 
-## [Unreleased]
+## [0.1.1]
+
+### Added
+
+- create-collection and matching tests for both glo-30 and glo-90
+- New: `--host` option to specify a Provider HOST from a list of options in constants. `AWS` and `OT` are the current valid options.
+- added python 3.10
+
+### Deprecated
+
+- Nothing
+
+### Removed
+
+- remove python 3.7 support
+
+### Fixed
+
+- Nothing.
+
+
+## [0.1.0]
```

Comment:
There's no tag for 0.1.0 right now, we should tag that if we want to be able to generate those diff links.

Reply:
Yeah, it looks like it wasn't ever released (https://pypi.org/project/stactools-cop-dem/ is a 404). In that case, this can all go in the `[Unreleased]` section, and we can make the CHANGELOG pretty before the first release.